### ISSUE TITLE
Home App status is "Scheduled" with no option to turn on

### ISSUE THREAD

**Issue by enz1ey** [MortJC/homebridge-platform-orbit#48]
I have removed any and all schedules from the B-Hyve app, and my Home app tile still just shows a status of "scheduled" and tapping on it doesn't give me any options to turn the controller on/off. It also doesn't update when the controller is turned on manually.

**Comment by enz1ey** [comment]
I just tested using the Eve app, and it seems I can control it there as expected. But then going back to the Home app, it did show the on/off toggle long enough for me to turn it on, but then it disappeared again and I was unable to turn it off without going back to the Eve app.

**Comment by enz1ey** [comment]
https://user-images.githubusercontent.com/6290727/167449406-83bf80b3-289b-4281-8e0a-dc3367844ed4.mov

Here you can see, tap and hold shows the control, then tapping quickly shows nothing. Another tap and hold and the controls are missing again. It's all over the place.

**Comment by enz1ey** [comment]
https://user-images.githubusercontent.com/6290727/167449737-231e63af-bab2-4ade-9bf1-45aa19350eac.mov

You can see here, a tap-and-hold shows the controls, while a quick tap shows nothing. Then another tap-and-hold and the controls are gone. It's all over the place.

**Comment by MoTechnicalities** [comment]
This is in fact an Apple Home App bug. Same occurs with true HomeKit enabled Yardian Pro and other name brand sprinkler system controllers.

**Comment by SpencerKaiser** [comment]
@MoTechnicalities any idea if there's a bug tracked for it or whether a fix will be out anytime soon? I'm just happy I saw this because I thought my setup was broken 😅 

@enz1ey in case you haven't tried, you can still control them by name with Siri: `Turn on the XXXXX sprinkler`

**Comment by MoTechnicalities** [comment]
@SpencerKaiser 
I reported it to Yardian Pro, then to Apple. I have verified that the new upcoming release of the Apple Home App will fix the problem. But evidently Apple has no intention fixing the current release. I can't imagine how upsets I would be if I where a manufacturer facing returned products from customer dissatisfaction over something Apple should take care of immediately!

**Comment by SpencerKaiser** [comment]
Yea jeez... boo on apple for not fixing it. Well thanks for the quick response! Looking forward to having it work again in the next release!

**Comment by MortJC** [comment]
Is this still an issue for you?

**Comment by SpencerKaiser** [comment]
@MortJC it's actually worse now... I can't even use Siri anymore; they're just totally bricked

**Comment by enz1ey** [comment]
Yeah, mine just shows “running schedule” and toggling the switch in HomeKit does nothing.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by MortJC** [comment]
@enz1ey Mine shows "Scheduled" on the iphone home. Where do you see "Running schedule"? And do you have anything scheduled?

**Comment by enz1ey** [comment]
![Screenshot 2022-10-12 at 4 03 49 PM](https://user-images.githubusercontent.com/6290727/195437253-e5b9aad1-131b-4951-a842-a38884137413.png)
I do have a program created which runs every day, but this shows all the time in the Home app, regardless of whether the program is running or not. The toggle button does nothing.

**Comment by MortJC** [comment]
Weird. Can you see me your debug log.
